File: chartiq/src/main/java/com/chartiq/sdk/model/AggregationType.java
package com.chartiq.sdk.model;

public enum AggregationType {
    HEIKINASHI("heikinashi"),
    KAGI("kagi"),
    RENKO("renko"),
    RANGEBARS("rangebars"),
    PANDF("pandf");

    private String value;

    AggregationType(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: chartiq/src/main/java/com/chartiq/sdk/model/AggregationType.kt
package com.chartiq.sdk.model

enum class AggregationType(val value: String) {
    HEIKINASHI("Heikin Ashi"),
    KAGI("Kagi"),
    LINEBREAK("Line Break"),
    RENKO("Renko"),
    RANGEBARS("Range Bars"),
    PANDF("Point & Figure"),
    OHLC("OHLC")
}
